### ISSUE TITLE
Nano ledger S - public adress display bug on device when using NEON wallet

### ISSUE THREAD

**Issue by madscyan** [CityOfZion/neon-wallet#1164]
**Wallet version or commit SHA:**
 0.2.4

**Environment:**
macOS high sierra 10.13.4

**Prerequisites:**
Access the neon wallet through the ledger nano S

**Reproduction steps:**
1: Unlock the neon wallet through the nano ledger S
2: Show the public adress on the neon wallet
3: On the nano ledger s, click on the button to display the public adress
Prerequisite: You'll need a public adress that has the same letter that i have or that displays the same problem.  (In this case a capital J as the last digit)

Description:
Refer to the image, you can clearly see that the last digit in my public adress is displayed differently on the physical device (nano s) than on my computer.  On the nano ledger s, the last 'J' in the public adress is displayed differently than the other 'J' in the middle of that same adress.  

I tried sending funds to test and 1 neo was transferred succesfully to that adress, making me think this is only a display bug.

![neon wallet display bug](https://user-images.githubusercontent.com/20310451/40676504-e58676a0-6348-11e8-88d7-b9b7cb9d8f4c.jpg)

I contacted nano ledger with this issue but they replied saying it was a neon wallet issue and basically deferred to you guys.   Other than the fix itself, i would like to know if this is just a display bug and i can ignore this, or if there is a possibility that my public adress on my nano ledger and the one displayed in the neon wallet app might be different.

Thanks

**Comment by madscyan** [comment]
Bump.  Waiting for a response please.

**Comment by drptbl** [comment]
It just looks like a cut "J" letter at the bottom on your Ledger display. Both addresses are the same.
